tests/test_taskgroups.py

```diff
@@ -884,3 +936,15 @@ async def exit_scope(scope):
     with pytest.raises(RuntimeError):
         async with create_task_group() as tg:
             tg.start_soon(exit_scope, scope)
+
+
+async def test_raised():
+    class SomeExc(Exception):
```

Comment:
You don't really need a custom exception here, do you?

Reply:
Changed to a `ValueError`